apps/studio/src/pages/_app.tsx

```diff
@@ -38,7 +39,15 @@ void gb.init({
   // Optional, enable streaming updates
   streaming: true,
 })
+
 const MyApp = ((props: AppPropsWithAuthAndLayout) => {
+  if (env.NEXT_PUBLIC_INTERCOM_APP_ID) {
+    Intercom({
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
this should not have an eslint error since the env should be typed properly

Reply:
yeah i tot so too but it's showing as an issue on my editor

<img width="569" alt="image" src="https://github.com/user-attachments/assets/f6265614-74cf-4cb7-a241-1ef2327f3782">

Reply:
what's the error? sometimes need restart eslint after adding env

Reply:
you are right - it was gone after i restart my code editor and restart eslint
thanks for pointing it out and sharing! :D